### PR TITLE
Added paragraph and link to home

### PR DESCRIPTION
**Pull request by stephlaf**
Please @Carms-dev  review this code and merge if satisfactory

### CODE REVIEW

File: app/views/pages/about.html.erb [hunk 1]
```diff
@@ -1 +1,5 @@
-<h1>About Us</h1>
+<h2>About Us</h2>
```
Comment (carms-ng) [inline app/views/pages/about.html.erb]: @stephlaf 
Each page should have at least one `<h1>` tag for SEO!

Comment (stephlaf) [reply app/views/pages/about.html.erb]: You're right. Let me change this!

File: app/views/pages/about.html.erb [hunk 2]
```diff
@@ -1 +1,5 @@
...
+
+<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Hic quo eos voluptatem est maiores facilis quae molestias dicta totam. Eum, debitis blanditiis placeat consectetur repudiandae accusantium a, accusamus cupiditate nobis.</p>
+
+<%= link_to 'Home', root_path %>
```
Comment (stephlaf) [inline app/views/pages/about.html.erb:5]: Should go to home?

Comment (carms-ng) [reply app/views/pages/about.html.erb:5]: YES!!!

**Pull request merged**